app/helpers/application_helper.rb

```diff
@@ -2,10 +2,22 @@
 
 module ApplicationHelper
   def profile_dashboard_url(user)
-    if user.admin_profile
+    if user.admin?
       admin_suppliers_url
+    elsif user.induction_coordinator?
+      induction_coordinator_dashboard_url(user)
     else
       dashboard_url
     end
   end
+
+private
+
+  def induction_coordinator_dashboard_url(user)
+    school = user.induction_coordinator_profile.schools.first
+
+    return schools_choose_programme_url unless school.chosen_programme?(Cohort.current)
+
+    schools_dashboard_url
```

Comment:
Are we sure this is the right behaviour? If there are multiple tasks, maybe showing them all on dashboard would be neater?

Ech, something to chase with designers I guess.

Reply:
The idea was you can't do anything until you've made this choice. I think in future when you have more than one cohort this might move under each of the cohorts on the dashboard (there's a post it on the wireframes for future iterations). But you can't see anything else until you've answered this, because the answer to this decides what you see